tasks/test_component_snapshot.yaml

```
@@ -35,8 +35,8 @@ spec:
           componentSha=$(echo "$components" | jq -r '.source.git.revision')
             # Check if component git url equals to snapshot git url, if yes check if the snapshot SHA equals to component SHA
             if [[ \"$componentUrlWithoutSuffix\" == $snapshotUrlUrlWithoutSuffix ]]; then
```

Note that this line was only wrong. I had to change it to:
```suggestion
            if [[ $componentUrlWithoutSuffix == $snapshotUrlUrlWithoutSuffix ]]; then
```

Commit: https://github.com/RedHatInsights/rhsm-subscriptions/commit/cf6925f48652abd1395bbfa0c685049a07fce363

Note that the rest of the changes in my commit is about to fix the indentation,